sftp/1.3/modules/ROOT/pages/sftp-list.adoc

```diff
@@ -30,88 +49,124 @@ The following example lists the contents of a folder and all messages in `direct
-path-pattern="a?*.{htm,html,pdf}"
-createdSince="2019-06-03T13:21:58+00:00"
-createdUntil="2019-07-03T13:21:58+00:00"
-updatedSince="2019-05-03T13:21:58+00:00"
-updatedUntil="2019-06-03T13:21:58+00:00"
-accessedSince="2019-06-03T13:21:58+00:00"
-accessedUntil="2019-06-03T13:21:58+00:00"
-directory="true|false"
-regularFile="true|false"
-symbolicLink="true|false"
-minSize="0"
-maxSize="1024" />
+  timestampSince="2019-06-03T13:21:58+00:00"
+  timestampUntil="2019-06-03T13:21:58+00:00"
+  timeUnit="MICROSECONDS"
+  filenamePattern="a?*.{htm,html,pdf}"
+  pathPattern="a?*.{htm,html,pdf}"
+  directories="REQUIRE"
+  regularFiles="REQUIRE"
+  symLinks="REQUIRE"
+  maxSize="1024"/>
+		</sftp:list>
 ----
 
-=== Individual Attributes
-
-Individual attributes are optional and are ignored if you do not provide values for them. Use an `AND` operator to combine individual attributes.
-
-* `accessedSince` +
-An inclusive lower boundary for the file access stamp expressed as either a `DateTime` instance or a string in ISO-8601 format.
-* `accessedUntil` +
-An inclusive upper boundary for the file access stamp expressed as either a `DateTime` instance or a string in ISO-8601 format.
-* `createdSince` +
-An inclusive lower boundary for the file creation stamp expressed as either a `DateTime` instance or a string in ISO-8601 format.
-* `createdUntil` +
-An inclusive upper boundary for the file creation stamp expressed as either a `DateTime` instance or a string in ISO-8601 format.
-* `directory` +
-Match only if the file is a directory.
-* `filenamePattern` +
-Similar to the current filename pattern filter but more powerful. Glob expressions (default) and regex are supported. You can select which one to use by setting a prefix: for example, `glob:*.{java, js}` or `regex:[0-9]test.csv`.
-* `maxSize` +
-Inclusive upper boundary for the file size expressed in bytes.
-* `minSize` +
-Inclusive lower boundary for the file size expressed in bytes.
-* `path-pattern` +
-Same as filenamePattern but applies over the entire file path rather than just a filename.
-* `regularFile` +
-Match only if the file is a regular file.
-* `symbolicLink` +
-Match only if the file is a symbolic link.
-* `updatedSince` +
-An inclusive lower boundary for the file modification stamp expressed as either a `DateTime` instance or a string in ISO-8601 format.
-* `updatedUntil` +
-An inclusive upper boundary for the file modification stamp expressed as either a `DateTime` instance or a string in ISO-8601 format.
-
-You can use the file matcher as either a named top level element, which allows it to be reused, or as an inner element that is proprietary to a particular component.
-
 === Top Level, Reusable Matcher
 
-This is an example of a top level, reusable matcher (not available in Flow Designer):
+You can use the file matcher as either a named top level element, which enables it to be reused, or as an inner element that is proprietary to a particular component.
```

Comment:
Suggestion:
You can use the file matcher as either a named top-level element that enables reuse or as an inner element that is proprietary to a particular component.